src/modules/hip/kernel/box_filter.hpp

```diff
@@ -293,7 +293,7 @@ __global__ void box_filter_3x3_pkd_tensor(T *srcPtr,
     int id_x_i = id_x_o - padLength;
     int id_y_i = id_y_o - padLength;
     d_float24 sum_f24;
-    __shared__ uchar src_lds[48][128];
+    __shared__ uchar srcShared[48][128];
```

Comment:
I prefer the current name src_lds. Also, based on the google coding standard, we should avoid the CamelCase for variable names. please see [here](https://google.github.io/styleguide/cppguide.html#Constant_Names:~:text=enum%20class%20UrlTableError%20%7B%20...-,Variable%20Names,-The%20names%20of). The only thing that I would suggest is to replace the 48, and 128 with the defines like the Gaussian filter PR.

Reply:
Hi @asalmanp :  When attending HIP training, they mentioned LDS is AMD's internal name so it is better not to use that name. Outside people doesn't really know what lds means. May I suggest src_shared[] instead?